backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxIteratorApi.scala

```diff
@@ -352,17 +354,32 @@ class VeloxIteratorApi extends IIteratorApi with Logging {
     new ArrowInIterator(delegated.asJava)
   }
 
+  /**
+   * Generate Memory ReservationListener.
+   *
+   * @return
+   */
+  override def genTaskMemoryResourceManager(taskMemoryManager: TaskMemoryManager,
```

Comment:
Would it be better to rename the API method name to `genNativeMemoryAllocatorManager`

Reply:
Fix